internal/controller/clusterorder_controller.go

```diff
@@ -64,10 +64,11 @@ func (r *ClusterOrderReconciler) components() []component {
 // ClusterOrderReconciler reconciles a ClusterOrder object
 type ClusterOrderReconciler struct {
 	client.Client
-	Scheme                *runtime.Scheme
-	CreateClusterWebhook  string
-	DeleteClusterWebhook  string
-	ClusterOrderNamespace string
+	Scheme                 *runtime.Scheme
+	CreateClusterWebhook   string
+	DeleteClusterWebhook   string
+	ClusterOrderNamespace  string
+	MinimumRequestInterval string
```

Comment:
Why isn't this a `time.Duration` variable (echoing @mhrivnak's comment elsewhere, why are we re-parsing this every time we call `triggerWebHook`)? And where does it get set? This structure is initialized by the `NewClusterOrderReconciler` method, but that method hasn't been updated to set this field.